crates/bevy_render/src/lib.rs

```diff
@@ -176,7 +180,8 @@ impl Plugin for RenderPlugin {
                 .insert_resource(adapter_info)
                 .insert_resource(pipeline_cache)
                 .insert_resource(asset_server)
-                .init_resource::<RenderGraph>();
+                .init_resource::<RenderGraph>()
+                .init_resource::<ReserveCount>();
```

Comment:
Update this variable name too <3

Reply:
Fixed in 409f644eff2ec32706b70574b926f856d2bf142c